src/pages/ProductInner/ProductInner.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+import { Preloader } from '../../components/Preloader/Preloader';
+import { ErrorMessage } from '../../components/ErrorMesage/ErrorMessage';
+
+import './ProductInner.scss';
+
+export function ProductInner() {
+  const { id } = useParams();
+  const [product, setPrdouct] = useState(null);
+  const [productImageUrl, setPdouctImageUrl] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    fetch(`https://api.thedogapi.com/v1/breeds/${id}`)
+      .then((res) => {
+        setLoading(false);
+        if (!res.ok) throw new Error();
+        return res.json();
+      })
+      .then((data) => setPrdouct(data))
+      .catch(() => setIsError(true));
+  }, []);
+
+  useEffect(() => {
+    product &&
+      fetch(`https://api.thedogapi.com/v1/images/${product.reference_image_id}`)
+        .then((res) => {
+          if (!res.ok) throw new Error();
+          return res.json();
+        })
+        .then((data) => setPdouctImageUrl(data.url))
+        .catch(() => setIsError(true));
+  }, [product]);
+
+  return (
+    <div className="ProductInner">
+      {loading && <Preloader />}
+      {isError && <ErrorMessage />}
+      {productImageUrl && (
```

Comment:
I think productImageUrl should only affect ProductInner__image-container, not the entire container

Imagine you get an error while performing get product image request - in this case all the information on this page will be empty regardless of the result the get product request